<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>579</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>769</width>
    <height>579</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>769</width>
    <height>579</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWid">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>580</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>771</width>
      <height>580</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>771</width>
      <height>850</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="sun_and_moon">
     <attribute name="title">
      <string>El Sol y La Luna</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>471</width>
        <height>487</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="button_sun">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>230</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>230</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>      Sol </string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/sun.png</normaloff>icons/sun.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>220</width>
           <height>220</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="button_moon">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>230</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>230</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>      Luna</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/moon.png</normaloff>icons/moon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>220</width>
           <height>220</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="the_planets">
     <attribute name="title">
      <string>Los Planetas</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>744</width>
        <height>536</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="button_mercury">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Mercurio</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/mercury.png</normaloff>icons/mercury.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="button_venus">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Venus   </string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/venus.png</normaloff>icons/venus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="button_earth">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Tierra  </string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/earth.png</normaloff>icons/earth.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="button_mars">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Marte   </string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/mars.png</normaloff>icons/mars.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="button_jupiter">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Jupiter </string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/jupiter.png</normaloff>icons/jupiter.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="4">
          <widget class="QPushButton" name="button_saturn">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Saturno </string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/saturn.png</normaloff>icons/saturn.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="button_uranus">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Urano   </string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/uranus.png</normaloff>icons/uranus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_neptune">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>160</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Neptuno </string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/neptune.png</normaloff>icons/neptune.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
